spec/javascripts/pageflow/ui/utils/i18n_utils_spec.js

```diff
@@ -31,6 +33,33 @@ describe('pageflow.i18nUtils', function() {
 
       expect(result).to.eq('Value interpolated');
     });
+
+    it('escapes html', function() {
+      var result = pageflow.i18nUtils.findTranslation(
+        ['html.key.without.suffix']
+      );
+
+      expect(result).to.eq('&lt;div /&gt;');
```

Comment:
I think this is the behavior that we would expect from Rails translations http://guides.rubyonrails.org/i18n.html#using-safe-html-translations

I agree on testing default escaping.